src/main/resources/static/styles.css

```diff
@@ -34,6 +34,70 @@
     margin: 2rem auto auto auto;
 }
 
+.garden-title {
+    text-align: center;
+    margin-top: 3rem;
+    margin-bottom: 1rem;
+    font-family: "Garamond";
+    font-weight: bold;
+    font-size: 225%;
+}
+
+.garden-subtitle {
+    text-align: center;
+    margin-bottom: 1rem;
+    margin-top: 1rem;
+    margin-left: 5rem;
+    margin-right: 5rem;
+    font-family: "Garamond";
+    font-weight: bold;
+    font-size: 160%;
+
+}
+
+.gardens {
+    background-color: #cae8d5;
+}
+
+.card {
+    border: none;
+    margin-top: 3rem;
+    margin-bottom: 3rem;
+}
+
+.card-img-top {
+    height: 20vw;
```

Comment:
Looks great, and functioned well for me! My only suggestion is the photos are still being cropped, but it you remove line 69 that should fix the issue.